### ISSUE TITLE
Drop UIWebView in favour of WKWebView

### ISSUE THREAD

**Issue by nicorsm** [Accengage/accengage-ios-sdk-releases#8]
Since Apple won't accept apps that makes use of `UIWebView` starting from December 2020, it would be good to replace `UIWebView` in the SDK with `WKWebView`. 
Right now, `A4SWebView` is a subclass of `UIWebView`, and this is causing an alert every time we submit a new version of an app to the App Store, even if we don't make use of `UIWebView` in our code.
This is the e-mail we receive from Apple:

> Dear Developer,
> 
> We identified one or more issues with a recent delivery for your app. Your delivery was successful, but you may wish to correct the following issues in your next delivery:
> 
> ITMS-90809: Deprecated API Usage - Apple will stop accepting submissions of apps that use UIWebView APIs starting from December 2020 . See https://developer.apple.com/documentation/uikit/uiwebview for more information.
> 
> After you’ve corrected the issues, you can upload a new binary to App Store Connect.
> 
> Best regards,
> 
> The App Store Team

**Comment by oristanovic** [comment]
We dropped support for UIWebView. You'll need to update the SDK, more info [here](https://documentation.accengage.com/sdk/changelog/ios)

**Comment by gudavo** [comment]
Hello,

We still have this issue with Accengage SDK 7.0.1, even though it should be resolved since 7.0.0 according to your changelog.
Apple keeps sending the warning email mentioned by @nicorsm.
Maybe you do not link `UIWebView` directly but even variables or classes names containing `UIWebView` can trigger Apple warning apparently.

Running `grep -r "UIWebView" ~/Downloads/Accengage.framework` produces the following output:
```
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-nobuttons-landscape.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-nobuttons-landscape.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-nobuttons-landscape-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-nobuttons-landscape-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-landscape.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-landscape.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-nobuttons-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-nobuttons-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-nobuttons-landscape.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-nobuttons-landscape.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-nobuttons.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-nobuttons.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-landscape-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-landscape-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-landscape-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-landscape-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-nobuttons-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-nobuttons-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-landscape.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-landscape.nib/runtime.nib matches
/Users/gdavo/Downloads/Accengage.framework/Headers/Accengage.h:    ACCUIWebViewFramework,
/Users/gdavo/Downloads/Accengage.framework/Headers/A4SWebView.h: You can use this webview as a standard UIWebView.
/Users/gdavo/Downloads/Accengage.framework/Headers/A4SWebView.h:@interface A4SWebView : UIWebView
/Users/gdavo/Downloads/Accengage.framework/Headers/BMA4SInAppNotificationDataSource.h: <td rowspan="8">__Web view__ : <b>UIWebView</b> with tag <i>BMA4SInAppOverlayViewBodyTag</i></td>
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-nobuttons-landscape.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-nobuttons-landscape.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-nobuttons.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-nobuttons.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-landscape.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-landscape.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/Accengage matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-nobuttons-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-nobuttons-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-landscape-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewNotifView-landscape-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-nobuttons.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-nobuttons.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-nobuttons-landscape-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneWebViewNotifView-nobuttons-landscape-iPad.nib/runtime.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-nobuttons-landscape-iPad.nib/objects-13.0+.nib matches
Binary file /Users/gdavo/Downloads/Accengage.framework/BMA4SiPhoneIntersticialWebViewWithToolBarNotifView-nobuttons-landscape-iPad.nib/runtime.nib matches
```
